graphics/libiptcdata/Portfile

```diff
@@ -24,6 +26,18 @@ master_sites        sourceforge:project/libiptcdata/libiptcdata/${version}
 checksums           rmd160  64e1b7cd1d5166091227eb651f9945bc3862e39b \
                     sha256  79f63b8ce71ee45cefd34efbb66e39a22101443f4060809b8fc29c5eebdcee0e
 
-depends_lib         port:gettext
+# Add missing intl-related libs to link path
+patchfiles-append   patch-Makefile.in-intl.diff
+
+depends_build-append \
+                    port:cctools \
+                    port:gettext \
+                    port:gtk-doc \
```

Comment:
Shouldn't the prebuilt docs in the distribution be sufficient, making a gtk-doc dep unnecessary?

Reply:
Good point, doc generation is disabled by default. I'll remove.